lib/cb/models/implementations/job.rb

```diff
@@ -156,7 +156,9 @@ def set_model_properties
         @description                  = args['Description'] || args['JobDescription'] || ''
         @requirements                 = args['JobRequirements'] || ''
         @begin_date                   = args['BeginDate'] || ''
+        @begin_date_time              = args['BeginDateTime'] || ''
```

Comment:
Good question.  The existing tests for this class only test the predicate methods.